js/whitebit.js

```diff
@@ -221,6 +223,11 @@ module.exports = class whitebit extends Exchange {
                     'margin': 'collateral',
                     'trade': 'spot',
                 },
+                'networksById': {
+                    'BEP20': 'BSC',
+                    'ERC20': 'ETH',
```

Comment:
@samgermain again please do not add these entries (trc20, erc20) here

Reply:
but we're keeping `BEP20 -> BSC` ?

Reply:
@samgermain tbh I'm sure what's the "correct" value for the BSC, currently, we have different notations (BSC/BEP20) across the board but yeah, for the sake of consistency we should probably remove them and add this entry to `defaultNetworkCodeReplacements`, wdyt? 
```
            'defaultNetworkCodeReplacements': {
                'BSC': { 'BEP20': 'BNB' },
```

Reply:
BNB corresponds to BEP2
BSC corresponds to BEP20